### PR TITLE
Initial scaffolding + listing of apprenticeships

### PR DESCRIPTION
**Pull request by benjaminmodayil**
I've done some initial scaffolding with the site. Also, I wrote a small one-time script to pull out each individual listing to its own separate markdown file, but I didn't include the script since it would only be used once to get the markdown files as they are currently in this repo.

In the initial issue #8 it was mentioned that there would probably be location and description frontmatter variables, but I quickly realized that there were others needed and I've included those as well. Let me know if you think otherwise.

I've also included the default "draft" variable in the frontmatter because I felt that it might be useful for setting up an automation of sorts if there were a form on the site to collect new ones. Obviously the form wouldn't immediately update the site because the listing would need approval, but I believe we might be able to setup a hook to automatically setup a PR with a form submission for approval.

example listing for IFTTT:
```
company: "IFTTT"
link: "https://ifttt.com/apprenticeship"
location: 
  - "San Francisco, CA"
description: "If This Then That, also known as IFTTT, is a free web-based service to connect all your apps and devices into one \"applet\"."
draft: false
```

Also, is this going to be hosted on Netlify like the tech stack listed? It would be helpful for preview links going forward, but you would have to set that up since you're the repository owner.
[EDIT: I just noticed the Netlify deploy bot below.]

I believe Netlify has free team functionality for open-source projects so it might be worth checking out. Or possibly overkill 🤷

https://www.netlify.com/open-source/

[EDIT: Should I add the `/public` folder to the `.gitignore` since I believe Netlify handles the build?

### CODE REVIEW

File: website/content/apprenticeships/8th-light.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+---
+company: "8th Light"
+link: "https://8thlight.com/apprenticeship/"
+location: 
+  - "London, UK"
+description: "8th Light is a software consultancy that is passionate about crafting beautiful and durable software products."
+draft: false
+---
+able software products."
```
Comment (fvcproductions) [inline website/content/apprenticeships/8th-light.md]: Looks like you got some extra stuff outside the front matter that can be removed

File: website/content/apprenticeships/accenture.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+---
+company: "Accenture"
+link: "https://www.accenture.com/gb-en/Careers/apprenticeships"
+location: 
+  - "London, UK"
+  - "Newcastle, UK"
+  - "Warwick, UK"
+description: "Accenture is a global professional services company providing a range of strategy, consulting, digital, technology & operations services and solutions."
+draft: false
```
Comment (fvcproductions) [inline website/content/apprenticeships/accenture.md]: We can take out the `draft: false` to make it cleaner for a lot of these.

File: .gitignore [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+*.ppt*
+
+# Backup files #
+#####################
+*.bak
+
+# Misc #
+#####################
+Icon^M^M
+Icon
```
Comment (fvcproductions) [inline .gitignore]: Can we put `public` in the `gitignore` as well. We actually don't want to host the output - we will let Netlify do that for us.

Comment (fvcproductions) [reply .gitignore]: Similar to how github.com/techqueria/website and github.com/fvcproductions/fvcproductions.github.io does it

File: website/config.toml [hunk 4]
```diff
@@ -0,0 +1,3 @@
+baseURL = "http://example.org/"
+languageCode = "en-us"
+title = "My New Hugo Site"
```
Comment (fvcproductions) [inline website/config.toml]: Let's change the `title` to "Apprenticeships" and `baseURL` to `https://apprenticeships.fvcproductions.com/`

**Comment by fvcproductions** [comment]
@benjaminmodayil Just sent you an invite so you can help with Netlify admin settings as well.

**Comment by benjaminmodayil** [comment]
@fvcproductions Awesome. I believe I integrated all your requested with the latest commit. I'll take a look at any Netlify setup needed. Also, I'll take a look at setting up a basic Algolia search. My focus so far is just getting any functionality out of the way. I was thinking that any styling/theming concerns could be treated last if that works with you.

**Comment by fvcproductions** [comment]
@benjaminmodayil That all sounds good to me!

**Pull request merged**